core/src/main/java/org/pentaho/platform/util/ActionUtil.java

```diff
@@ -91,6 +91,27 @@
     KEY_MAP.put( ActionUtil.QUARTZ_RESTART_FLAG, ActionUtil.INVOKER_RESTART_FLAG );
   }
 
+  public static final String WORK_ITEM_XMS = "JVM_XMS";
+  public static final String WORK_ITEM_XMX = "JVM_XMX";
+
+  /**
+   * Validates a JVM_XMS or JVM_XMX value to make sure it has the correct format.
+   * In case the format is not correct, it uses the default value.
+   *
+   */
+  public static boolean validateJvmXm( String jvmXmVar ) {
+    try {
+      int intValue = Integer.parseInt( jvmXmVar );
```

Comment:
Have we considered commons-validator to do this work for us? 

https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/IntegerValidator.html#validate(java.lang.String)

Reply:
Will change to use the IntegerValidator